docker/Dockerfile.hipcc

```diff
@@ -89,7 +89,7 @@ RUN SCRATCH_DIR=/scratch && mkdir -p ${SCRATCH_DIR} && cd ${SCRATCH_DIR} && \
     rm -rf ${SCRATCH_DIR}
 
 # Install Kokkos
-ARG KOKKOS_VERSION=4.0.00
```

Comment:
Why not bump `4.0.00` to `4.1.00` and `4.1.00` to `4.2.00`?

Reply:
I thought we would leap frog the versions. Change 4.0 to 4.2 now, change 4.1 to 4.3 next time. This way we have a mix of older builds and newer builds.